### PR TITLE
Update UG with edit and search features

### LINKED ISSUE #3: Update the User Guide

### PR DESCRIPTION
**Pull request by Persdre**

### CODE REVIEW

**Comment by Persdre** [comment]
for #3

File: docs/UserGuide.md [hunk 1]
```diff
@@ -89,9 +89,9 @@ Format: `list`
 
 ### Editing a person : `edit`
 
-Edits an existing person in the address book.
+Edits the corresponding contact information in the contact list.
 
-Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]…​`
```
Comment (yanbingtao) [inline docs/UserGuide.md:94]: I think we should still use this prefix ("n/", "p/", etc), to distinguish the input is either a name or a tag or something else.

File: docs/AboutUs.md [hunk 2]
```diff
@@ -38,15 +38,15 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
...
+<img src="images/wangqian.jpg" width="200px">
 
-[[github](http://github.com/johndoe)]
-[[portfolio](team/johndoe.md)]
+[[github](http://github.com/persdre)]
+[[portfolio](team/wangqian.md)]
 
-* Role: Developer
-* Responsibilities: Dev Ops + Threading
+* Role: Code Quality Reviewer
```
Comment (ureshiiYing) [inline docs/AboutUs.md:48]: I think you should keep "Developer".

**Review by ureshiiYing — COMMENTED** [review]
Refer to my other review.

File: docs/UserGuide.md [hunk 3]
```diff
@@ -104,11 +104,11 @@ Examples:
...
+### Find persons by keywords: `search`
 
-Finds persons whose names contain any of the given keywords.
+Finds all contacts that contain the KEYWORD(s)
 
-Format: `find KEYWORD [MORE_KEYWORDS]`
+Format: `search KEYWORD [MORE_KEYWORDS]`
 
 * The search is case-insensitive. e.g `hans` will match `Hans`
 * The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`
```
Comment (ureshiiYing) [inline docs/UserGuide.md:114]: I checked again, and I think you missed out the find sales history feature under sales tracking.

**Review by ureshiiYing — COMMENTED** [review]
To add the "Find the sales history data based on date" feature into UG

File: docs/UserGuide.md (hunk 3 above)
Comment (Persdre) [reply docs/UserGuide.md:114]: Yes, I was not sure about the format so I didn't update it.

File: docs/UserGuide.md (hunk 1 above)
Comment (Persdre) [reply docs/UserGuide.md:94]: Thank you! I already updated it!

**Pull request merged**